src/main/java/com/enderio/base/data/recipe/MaterialRecipes.java

```diff
@@ -83,8 +81,12 @@ protected void buildCraftingRecipes(Consumer<FinishedRecipe> recipeConsumer) {
         makeMaterialRecipes(recipeConsumer, EIOItems.DARK_STEEL_INGOT.get(), EIOItems.DARK_STEEL_NUGGET.get(), EIOBlocks.DARK_STEEL_BLOCK.get());
         makeMaterialRecipes(recipeConsumer, EIOItems.SOULARIUM_INGOT.get(), EIOItems.SOULARIUM_NUGGET.get(), EIOBlocks.SOULARIUM_BLOCK.get());
         makeMaterialRecipes(recipeConsumer, EIOItems.END_STEEL_INGOT.get(), EIOItems.END_STEEL_NUGGET.get(), EIOBlocks.END_STEEL_BLOCK.get());
-        
-        ShapedRecipeBuilder.shaped(EIOItems.CONDUIT_BINDER_COMPOSITE.get())
+
+        SimpleCookingRecipeBuilder.smelting(Ingredient.of(EIOItems.CONDUIT_BINDER_COMPOSITE.get()), EIOItems.CONDUIT_BINDER.get(), 0.0F, 200)
+            .unlockedBy("has_ingredient", has(EIOItems.CONDUIT_BINDER_COMPOSITE.get()))
+            .save(recipeConsumer);
```

Comment:
Forge Supports Multiple Outputs for Furnace Recipes, could you create a cookingRecipeBuilder that allows multiple outputs by extending the SimpleCookingRecipeBuilder and add a count field to it and serialize the result as a jsonobject, so this behaves similar to 1.12?

Reply:
also make this a smelting and a blasting recipe